### DISCUSSION TITLE
Trouble downloading from site

### DISCUSSION THREAD

**Discussion by Starfiretbt** [aclap-dev/video-downloadhelper#457] [Help and Q&A]
### Your question

I'm having trouble downloading movies and television shows from the site "movie4kto.net" with the app "video downloadhelper." Sometimes the movie or episode downloads but sometimes it seems to get stuck. The time till completion would be normal and the percentage of video downloaded would progress at a normal pace but then the percentage of completion would stay at a fixed percentage and the time till completion would just get longer and longer without any sign the video is downloading.

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:125.0) Gecko/20100101 Firefox/125.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\TBTol",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by Mattmjs** [comment]
Could just be the site, I see that download speed is all over the place sometimes dropping to 0 Mbps in the task manager.
There is no consistent speed, I tried the vidcloud server but there was always an error on the video player.

**Reply by Starfiretbt** [reply]
Thank you for your response, if there is nothing the extension can do about that because it is entirely the site's problem then that is a disappointment.

**Reply by Starfiretbt** [reply]
The new version of this extension has a feature to "copy bug details" does this data show what is happening on these sites and is there a solution?
{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://123moviesfree-ma.123caches.re/watch-series/married-with-children-season-9-full-episodes-online-free/123movies-ypw0env6-vype5alp?watchnow=21",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [https @ 000001afdbfbb280] Stream ends prematurely at 483029, should be 583928\r\n[aac_adtstoasc @ 000001afdc40d100] Error parsing ADTS frame header!\r\n[mp4 @ 000001afdbfa3cc0] Error applying bitstream filters to an output packet for stream #1: Invalid data found when processing input\r\n[aost#0:1/copy @ 000001afdcb126c0] Error submitting a packet to the muxer: Invalid data found when processing input\r\n[out#0/mp4 @ 000001afdc321a00] Error muxing a packet\r\n",
    "id": "coapp_failure_62c395ce-f7be-4cfd-bc45-a6603b0c0b55",
    "downloadable_id": "downloadable_8687112328950552"
  }
}

**Reply by Mattmjs** [reply]
I didn't have an issue on that one, it downloaded all the way to the end. I have pretty much an identical setup as you. 
The vidcloud sever didn't work. I then used SEmbed sever. Not sure why it failed for you.

**Reply by Starfiretbt** [reply]
It works 70-90% of the time. I think it fails when my internet becomes limited, such as during peak consumption or when my roommate is playing internet intensive games.
